code/_globalvars/mapping.dm

```diff
@@ -54,3 +54,7 @@ GLOBAL_LIST_EMPTY(space_ruins_templates)
 GLOBAL_LIST_EMPTY(lava_ruins_templates)
 GLOBAL_LIST_EMPTY(shelter_templates)
 GLOBAL_LIST_EMPTY(shuttle_templates)
+
+// Teleport locations
+GLOBAL_LIST_EMPTY(teleportlocs)
+GLOBAL_LIST_EMPTY(ghostteleportlocs)
```

Comment:
what a horrible names. Can you make these snake_cased?

Reply:
No, because I dont want to touch excess files in this PR. I want this to be as easy as possible to review without changing about 30 glob references